api/output_policy.go

```diff
@@ -0,0 +1,197 @@
+package api
```

Comment:
Looking at this file overall, it seems like it is using client rather than is a part of client. Would it be more appropriate to have it reside inside `command/` instead?

Reply:
(see other resolved comment)